### PR TITLE
added str-repeat.md in str-repeat folder in the string functions in php

### PR DESCRIPTION
**Pull request by MMMMMMMZ**
- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/string-functions/terms/str-repeat/str-repeat
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/str-repeat/str-repeat:2]:
```suggestion
Title: 'str_repeat()'
```

File: content/php/concepts/string-functions/terms/str-repeat/str-repeat
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/str-repeat/str-repeat:3]:
```suggestion
Description: 'Returns a string with occurrences of a specified substring repeated a specified number of times.'
```

File: content/php/concepts/string-functions/terms/str-repeat/str-repeat
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/str-repeat/str-repeat:16]: Delete the repeated metadata.

File: content/php/concepts/string-functions/terms/str-repeat/str-repeat
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/str-repeat/str-repeat:31]: Delete this header. The metadata Title key will provide the title for the entry. Once you delete this and the repeated metadata, make sure there's only one blank line between the end of the metadata and the first line of the entry.

File: content/php/concepts/string-functions/terms/str-repeat/str-repeat
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/str-repeat/str-repeat:33]:
```suggestion
The `str_repeat()` function returns a string that repeats a substring a specified number of times.
```

File: content/php/concepts/string-functions/terms/str-repeat/str-repeat
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/str-repeat/str-repeat:41]:
```suggestion
- `$string`:  Specifies the string to repeat and it is required.
```

File: content/php/concepts/string-functions/terms/str-repeat/str-repeat
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/str-repeat/str-repeat:42]:
```suggestion
- `$repeat`: Specifies the number of times the string will be repeated. Must be greater or equal to 0 and it is also required.
```

File: content/php/concepts/string-functions/terms/str-repeat/str-repeat
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/str-repeat/str-repeat:44]:
```suggestion
## Codebyte Example
```

File: content/php/concepts/string-functions/terms/str-repeat/str-repeat
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/str-repeat/str-repeat:46]:
```suggestion
This example provides a few demonstrations of `str_repeat()` operating on a string.

```

File: content/php/concepts/string-functions/terms/str-repeat/str-repeat
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/str-repeat/str-repeat:56]:
```suggestion

Notice how the third `echo` statement doesn't appear in the output due it being repeated zero times.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@MMMMMMMZ I did a first review and left a few comments.

File: content/php/concepts/string-functions/terms/str-repeat/str-repeat
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/str-repeat/str-repeat:24]:
```suggestion
- `$string`: Specifies the string to repeat and it is required.
```

**Review by SSwiniarski — APPROVED** [review]
@MMMMMMMZ Aside from one extra space, looks good for a second review.

**Comment by MMMMMMMZ** [comment]
ok, is it ready now for the second review?

File: content/php/concepts/string-functions/terms/str-repeat/str-repeat
Comment (KTom101) [inline content/php/concepts/string-functions/terms/str-repeat/str-repeat:36]:
```suggestion
echo (str_repeat("I-don't-appear ",0). "\n");
```

File: content/php/concepts/string-functions/terms/str-repeat/str-repeat
Comment (KTom101) [inline content/php/concepts/string-functions/terms/str-repeat/str-repeat:41]: Bump this line up to line 30 so the codebyte is the last thing on the page.
```suggestion
```

File: content/php/concepts/string-functions/terms/str-repeat/str-repeat
Comment (KTom101) [inline content/php/concepts/string-functions/terms/str-repeat/str-repeat:30]:
```suggestion

> **Note:** The third echo statement will not output to the console due to the second argument of `str_repeat` being set to `0`.

```

File: content/php/concepts/string-functions/terms/str-repeat/str-repeat
Comment (KTom101) [inline content/php/concepts/string-functions/terms/str-repeat/str-repeat:35]:
```suggestion
echo (str_repeat("Cool ",10). "\n");
```

File: content/php/concepts/string-functions/terms/str-repeat/str-repeat
Comment (KTom101) [inline content/php/concepts/string-functions/terms/str-repeat/str-repeat:34]:
```suggestion
echo (str_repeat("I-am-repeated ",3)."\n");
```

**Review by KTom101 — CHANGES_REQUESTED** [review]
@MMMMMMMZ I've added a second review.

**Comment by SSwiniarski** [comment]
@MMMMMMMZ I did a fix to the filename (it was missing the ".md") and ran an `npm format` so the linting checks would pass. This should be ready to merge once @KTom101 reviews again and approves your edits.

**Comment by KTom101** [comment]
@MMMMMMMZ Live link is here: https://www.codecademy.com/resources/docs/php/string-functions/str-repeat

Currently, you're listed as an anonymous contributor:

Screenshot 2023-05-30 at 12 21 04 PM

If you want to remain anonymous, there is nothing more you need to do. If you wish to be listed as a contributor, make sure you've added your GitHub username to your Codecademy profile using the following steps:

1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:

YourGitHubName

2.) Then log out and log back into Codecademy using your GitHub login:

ReLogIn

> **Note:** Your email used for Codecademy will have to be listed in your GitHub profile for the login feature to properly work. If you use the same email for both GitHub and Codecademy already, then you're set.

cc @yangc95   @SSwiniarski

**Pull request merged**
